src/config/networks/network.d.ts

```diff
@@ -0,0 +1,59 @@
+import { ETHEREUM_NETWORK } from 'src/logic/wallets/getWeb3'
+
+type DisplayFlag = 'enable' | 'disable'
+
+// matches src/logic/tokens/store/model/token.ts `TokenProps` type
+type Token = {
+  address: string
+  name: string
+  symbol: string
+  decimals: number
+  logoUri?: string
+}
+
+type NetworkSettings = {
+  id: ETHEREUM_NETWORK,
+  color: string,
+  label: string,
+  nativeCoin: Token,
+}
+
+// something around this to display or not some critical sections in the app, depending on the network support
+// I listed the ones that may conflict with the network.
+// If non is present, all the sections are available.
+type SafeFeatures = {
+  safeApps?: DisplayFlag,
```

Comment:
can we use a boolean instead? in order to simplify things

Reply:
I think in this case using a boolean can also be explicit enough
```
{
  features: {
    safeApps: true
  }
}
```

```
{
  features: {
    safeApps: 'enabled'  }
}
```